examples/nas/classic_nas/mnist.py

```diff
@@ -0,0 +1,181 @@
+"""
+A deep MNIST classifier using convolutional layers.
+
+This file is a modification of the official pytorch mnist example:
+https://github.com/pytorch/examples/blob/master/mnist/main.py
+"""
+
+import os
+import argparse
+import logging
+import nni
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+
+from nni.nas.pytorch.mutables import LayerChoice, InputChoice
+from nni.nas.pytorch.classic_nas import get_apply_next_architecture
+
+
+logger = logging.getLogger('mnist_AutoML')
```

Comment:
Not sure if this name of logger works, as it's not under nni name space. It should be double checked.

Reply:
checked, no problem here